risk-to-severity-mapping/README.md

```diff
@@ -0,0 +1,59 @@
+# Script to Map Severities to Risk Scores
```

Comment:
\+ add the new tool to the list on the readme at the root of the repo https://github.com/GitGuardian/ggtools/blob/main/README.md